internal/pkg/template/env.go

```diff
@@ -94,6 +94,12 @@ var (
 	}
 )
 
+// Addons holds data about an aggregated addons stack.
+type Addons struct {
```

Comment:
Does this only apply to env addons or any addons?

Reply:
This is generic enough for any addons. For now it's only used for env addons.